<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CwmsGroupWidget</class>
 <widget class="QWidget" name="CwmsGroupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>WMS Gruppeneditor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel10">
       <property name="text">
        <string>Gruppen ID</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_pqleGroupName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="m_pqleGroupId">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="textLabel11">
       <property name="text">
        <string>Gruppenname</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Gruppenmitglieder</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QTreeWidget" name="m_pqlvGroupMembers">
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Vorname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Nachname</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Login</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="m_pqpbAdd">
       <property name="toolTip">
        <string>Benutzer zur Gruppe hinzufÃ¼gen</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/usermanagerresources.qrc">
         <normaloff>:/png/user-add.png</normaloff>:/png/user-add.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbRemove">
       <property name="toolTip">
        <string>Benutzer aus Gruppe entfernen</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/usermanagerresources.qrc">
         <normaloff>:/png/user-delete.png</normaloff>:/png/user-delete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pqpbUpdate">
       <property name="toolTip">
        <string>Aktualisieren</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/usermanagerresources.qrc">
         <normaloff>:/png/ok.png</normaloff>:/png/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources>
  <include location="../resources/usermanagerresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_pqpbUpdate</sender>
   <signal>clicked()</signal>
   <receiver>CwmsGroupWidget</receiver>
   <slot>UpdateClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbAdd</sender>
   <signal>clicked()</signal>
   <receiver>CwmsGroupWidget</receiver>
   <slot>AddClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pqpbRemove</sender>
   <signal>clicked()</signal>
   <receiver>CwmsGroupWidget</receiver>
   <slot>RemoveClickedSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
